google-cloud-logging/src/main/java/com/google/cloud/logging/Severity.java

```diff
@@ -26,6 +26,9 @@
  */
 public enum Severity {
 
+  /** The unrecognized severity level. Should not be used with log entries. */
+  UNRECOGNIZED(LogSeverity.UNRECOGNIZED),
```

Comment:
there is already an existing severity for the kind of "undefined/unrecognized" type of severity which is `Severity.DEFAULT`.

Reply:
> Given the expectations to the backward compatibility and an attempt to avoid introduction of the new interfaces while keeping the existing interface intuitive, I propose the following implementation and recommend to extend it to the java-logging-logback library:
> 
> 1. Refactor `LoggingHandler.severityFor(Level level)` method to switch/case based on the `java.util.logging.Level` values and not `Integer` values that supposed to reflect the numeric values of the logging levels.
> 2. For the `case Level.OFF` return null:
>    ```java
>    case Level.OFF:
>        return null;
>    ```
> 3. Keep the rest implementation (I mainly reference to [LoggingImpl.java:885](https://github.com/googleapis/java-logging/blob/34b20a277d22174907329a900918e9a24dbdd8de/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java#L885)) as it is now.

This is actually going to break backward compatibility since returning `null` from this method never happened before...

Reply:
> there is already an existing severity for the kind of "undefined/unrecognized" type of severity which is `Severity.DEFAULT`.

`Severity.DEFAULT` is recognized today by Logging as appropriate level and considered as default, while `LogSeverity.UNRECOGNIZED` is not.

Reply:
It seems that `Level` is a class and I am planning to refactor `LoggingHandler.severityFor(Level level)` to work for ranges